src/core/Akka/Configuration/FlatConfig.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Text;
+using Hocon;
+
+namespace Akka.Configuration
+{
+    public class FlatConfig:Config
```

Comment:
Do we want to mark this as `internal` and `sealed`?

Reply:
I think `sealed` is better, yes